app/queries/content_change_matcher.rb

```diff
@@ -0,0 +1,7 @@
+class ContentChangeMatcher
+  def self.call(subscription:)
```

Comment:
Just been thinking a bit further about this and I'm not sure it will work as is. Won't this return all the content changes that ever matched the subscriber lists? Also we might need to call this with an array of subscriptions if we're building one digest per subscriber.

Reply:
It will return all content changes, I assumed it would be up to the caller to limit the results by the date range they want for the digest. 

I believe the decision has been made to stick with building one digest per subscription rather than per subscriber, so we shouldn't need to pass in an array.